courses/linux_basics/command_line_basics.md

```diff
@@ -326,11 +326,14 @@ More command displays the contents of a file or a command output,
 displaying one screen at a time in case the file is large (Eg: log files).
 It also allows forward navigation and limited backward navigation in the file.
 
+![](images/linux/commands/image10.png)
```

Comment:
```suggestion
![](images/linux/commands/image33.png)
```

Reply:
@codophobia I guess you mistyped the image number here.

Reply:
@ritwik12 That image was added twice by mistake. I have corrected the image number also. 👍